docs/requirements/requirements.md

```diff
@@ -0,0 +1,103 @@
+# Hardware requirements of TwPM module
+
+## Acronyms
+
+* Legend:
+* PTP – Platform TPM Profile
+* CRB – Command Response Buffer interface
+* DDWG – Device Driver’s Writers Guide
+* Certification PP – Certification Protection Profile
+* TIS – TPM Interface Specification
+
+## Requirements
+
+### Power supply
+
+#### Power supply voltage
+
+1. TPM MUST support a supply voltage of 1.8V or 3.3V.
+2. TPM MAY support supply and I/O voltages of both 1.8V and 3.3V.
+3. TPM MAY support other supply voltages.
+
+Source: [tcg_client_v13r27] l.1373-1375
+
+#### Power supply max current
+
+Maximum power supply taken by TPM MUST be 250mA.
+
+### SPI Interface
+
+#### Frequency
+
+Default clock frequency for PC Client platforms is defined to be 24MHz.
+Clock frequency signal, minimum, is 10MHz.
+
+Source: [tcg_client_v13r27] Chapter 6.4.1, line 1331
+
+#### Voltage levels
+
+1. TPM MUST support a supply and I/O voltage of 1.8V or 3.3V
+2. TPM MAY support supply and I/O voltages of both 1.8 and 3.3V.
+3. TPM MAY support other supply and I/O voltages. 
+
+Source: [tcg_client_v13r27] l.1373-1375
+
+#### Power supply
+
+
+#### Power sequence
+
+  * TPM_Init (LRESET#/SPI_RST#) signal connected to platform CPU Reset signa
+  * TPM main power pins (3V) must be connected such that the TPM is powered
+  during ACPI states S0-S2 and may be powered in S3-S5
+  * Optional implementation VBAT and/or 3VSB pins
+
+Source: [tcg_client_v13r27], Chapter 6.7.2
+
+#### Command Duration
```

Comment:
Command duration and timeouts don't depend on SPI, they should be moved to a separate category.

Reply:
(corrected in next commit)